<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClangTidyPreferences</class>
 <widget class="QWidget" name="ClangTidyPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="settingsLayout" stretch="0,0,1">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="pathsGroupBox">
     <property name="title">
      <string comment="@title:group">Paths</string>
     </property>
     <layout class="QHBoxLayout" name="pathsGroupBoxLayout">
      <item>
       <layout class="QVBoxLayout" name="pathsLabelsLayout">
        <item>
         <widget class="QLabel" name="clangtidyLabel">
          <property name="text">
           <string comment="@label:chooser">Clang-&amp;Tidy executable:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>kcfg_clangtidyPath</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="pathsUrlsLayout">
        <item>
         <widget class="KUrlRequester" name="kcfg_clangtidyPath">
          <property name="toolTip">
           <string comment="@info:tooltip">The full path to the Clang-Tidy executable</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="jobsBox">
     <property name="title">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="kcfg_parallelJobsEnabled">
        <property name="text">
         <string comment="@option:check">Run analysis jobs in parallel</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="kcfg_parallelJobsAutoCount">
          <property name="text">
           <string comment="@option:check">Use all CPU cores</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="parallelJobsFixedCountLabel">
          <property name="text">
           <string comment="@label:spinbox">Maximum number of threads:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="kcfg_parallelJobsFixedCount">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ClangTidy::CheckSetManageWidget" name="checksets" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ClangTidy::CheckSetManageWidget</class>
   <extends>QWidget</extends>
   <header>config/checksetmanagewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
